include/aws/http/proxy_strategy.h

```diff
@@ -228,23 +212,27 @@ struct aws_http_proxy_strategy_factory_tunneling_kerberos_options {
 
 struct aws_http_proxy_strategy_factory_tunneling_adaptive_kerberos_options {
     struct aws_http_proxy_strategy_factory_tunneling_kerberos_options kerberos_options;
-    
+ };
 
-};
+/*SA-Added Start*/
+ struct aws_http_proxy_strategy_factory_kerberos_auth_config {
+
+     /* type of proxy connection being established, must be forwarding or tunnel */
+     enum aws_http_proxy_connection_type proxy_connection_type;
+
+     /* user token to use in kerberos authentication which is base64 encoded and provided by user*/
```

Comment:
What is the natural format for the likely kerberos api/impl that retrieves the token?  If it's binary we should take binary and do the base64 ourselves, but if it's likely to be base64 already then this is fine.

Reply:
The api's that will retrieve the token for kerberos OR challenge/response for NTLM will be base64 encoded

Reply:
The user/customer is already provided base64 encoded token/response/challenge